app/routes.py

```diff
@@ -1,2 +1,93 @@
-from flask import Blueprint
+import re
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.Planet import Planet
```

Comment:
something to consider: the file itself should be lowercase as in `planet.py` whereas the class itself is uppercase